components/ai_chat/resources/untrusted_conversation_frame/untrusted_conversation_frame.tsx

```diff
@@ -9,13 +9,20 @@ import { setIconBasePath } from '@brave/leo/react/icon'
 import '$web-components/app.global.scss'
 import '$web-common/defaultTrustedTypesPolicy'
 import ConversationEntries from './components/conversation_entries'
-import { UntrustedConversationContextProvider } from './untrusted_conversation_context'
+import {
+  UntrustedConversationContextProvider
+} from './untrusted_conversation_context'
+import {
+  useUntrustedFrameDragHandling
+} from './hooks/useUntrustedFrameDragHandling'
 
 import '../common/strings'
 
 setIconBasePath('chrome-untrusted://resources/brave-icons')
 
 function App() {
+  useUntrustedFrameDragHandling()
```

Comment:
This doesn't need to be a hook as it doesn't do anything react-related. We could just call it outside of the App function, in the module root. And remove the `React.useEffect` wrapper and rename to something like `untrustedFrameDragHandlingSetup()`?

Reply:
Fixed in 35035cb2be6c06c49511a0049560478c44364368